nextflow_schema.json

```diff
@@ -502,6 +502,18 @@
                     "fa_icon": "fas fa-address-card",
                     "help_text": "By default mOTUs will report species names rather than NCBI Taxon IDs. In nf-core/taxprofiler we prefer taxon IDs due to interoperatbility and comparability with the output of other classifiers and profilers. If you prefer to have just species names, you can specify this to remove the IDs.\n\n> Modifies tool parameter(s):\n- mOTUs: `-p` (removed from the default nf-core/taxprofiler command)"
                 },
+                "run_kmcp": {
+                    "type": "boolean",
+                    "description": "Turn on classification with kmcp.",
+                    "fa_icon": "fas fa-toggle-on"
+                },
+                "kmcp_mode": {
+                    "type": "integer",
+                    "default": 3,
+                    "description": "Profiling mode.",
+                    "help_text": "Available values: \n0 (for pathogen detection)\n1 (higherrecall)\n2 (high recall)\n3 (default)\n4 (high precision)\n5 (higher precision)",
```

Comment:
Needs the 'Modifies tool(s) parameters' thing we have in other non `--run_XYZ` parameters

Reply:
Done